BitwardenShared/UI/Auth/Extensions/AuthRouter+Redirects.swift

```diff
@@ -31,6 +31,12 @@ extension AuthRouter {
         if account.profile.forcePasswordResetReason != nil {
             return .updateMasterPassword
         } else {
+            let isCarouselEnabled: Bool = await services.configService.getFeatureFlag(.nativeCarouselFlow)
+            let introCarouselShown = await services.stateService.getIntroCarouselShown()
+            if isCarouselEnabled, !introCarouselShown {
+                await services.stateService.setIntroCarouselShown(true)
```

Comment:
🤔 If I'm understanding this correctly, we will also only show the carousel the first time the app is launched. So even if someone doesn't log in or create an account, they won't see it a second time. Just want to make sure this is intended behavior, as I feel the ticket is a bit ambiguous on this; and the PR summary indicates that it's only keyed to having accounts on the device, implying that it should pop up each time until a user successfully logs in or creates an account.

Reply:
>  we will also only show the carousel the first time the app is launched. So even if someone doesn't log in or create an account, they won't see it a second time

The `completeAuthRedirect()` method is only called once someone has successfully logged in or created an account right before the vault is shown. So if the app is launched and you see the carousel, but you don't log in or create an account, we wouldn't get here and then you would still see it on subsequent app launches until an account is added.

Reply:
Makes sense, was just making sure!